uniffi_bindgen/src/bindings/python/templates/wrapper.py

```diff
@@ -20,8 +20,8 @@
 import contextlib
 
 {% include "RustBufferTemplate.py" %}
-
-{% include "RustBufferHelper.py" %}
```

Comment:
I split this up into two files, one for the reading and one for the writing. It would have been nice to group the operations by datatype rather than read/write, but didn't really work out with the structure of the abstractions here.